pkg/kubectl/cmd/get.go

```diff
@@ -212,6 +215,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			FilenameParam(enforceNamespace, &options.FilenameOptions).
 			SelectorParam(selector).
 			ExportParam(export).
+			IncludeUninitialized(includeUninitialized).
```

Comment:
`kubectl get -w pods foo` should include the events of uninitialized `foo`. 
`kubectl get -w pods` doesn't include uninitialized pods unless `--include-uninitialized=true`.